src/components/Exchange/FeesTooltip.tsx

```diff
@@ -68,46 +68,55 @@ function getFeesRows(isOpening: boolean, formattedFees: Record<string, string>)
   return rows.filter((row) => row.value);
 }
 
+function getTotalFees(fees: (BigNumberish | undefined)[]) {
+  return fees.reduce((acc: BigNumber, fee) => {
+    if (fee instanceof BigNumber) {
+      return acc.add(fee);
+    }
+    return acc;
+  }, BigNumber.from(0));
+}
+
 type Props = {
-  totalFees: BigNumber;
   executionFees: ExecutionFee;
   positionFee?: BigNumber;
   depositFee?: BigNumber;
   swapFee?: BigNumber;
-  fundingFee?: string;
+  fundingFee?: BigNumber | string;
   isOpening?: boolean;
+  titleText?: string;
 };
 
 function FeesTooltip({
-  totalFees,
   fundingFee,
   positionFee,
   swapFee,
   executionFees,
   depositFee,
   isOpening = true,
+  titleText = "",
 }: Props) {
   const { chainId } = useChainId();
   const executionFee = executionFees?.fee;
   const executionFeeUSD = executionFees?.feeUSD;
-
   const formattedFees = {
     swap: getFeesStr(swapFee),
     position: getFeesStr(positionFee),
     deposit: getFeesStr(depositFee),
     execution: getExecutionFeeStr(chainId, executionFee, executionFeeUSD),
-    funding: fundingFee || "",
+    funding: fundingFee instanceof BigNumber ? getFeesStr(fundingFee) : fundingFee,
```

Comment:
is there a chance `fundingFee` is not undefined and not BigNumber? thinking if it's simpler to just check if it's truthy (i.e. `fundingFee ? getFeesStr(fundingFee) : fundingFee`)

Reply:
Updated!